Plugins/MongoDataStore/server/tests/configuration/getConfigurationResponse.json

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
Duplicate of Burpsuite plugin. Please remove

Reply:
will do